modules/fedimint-mint-client/src/lib.rs

```diff
@@ -154,6 +155,44 @@ impl OOBNotes {
             .expect("Invariant violated: OOBNotes does not contain any notes")
     }
 
+    pub fn notes_json(&self) -> Result<serde_json::Value, serde_json::Error> {
+        let mut notes_map = serde_json::Map::new();
+        for notes in self.0.iter() {
+            match notes {
+                OOBNotesData::Notes(notes) => {
+                    let notes_json = serde_json::to_value(notes).expect("serialization can't fail");
```

Comment:
Nit: the other `to_value` return `Result` rather than `expect`, you could do the same here